code/modules/surgery/mechanic_steps.dm

```diff
@@ -0,0 +1,85 @@
+//open shell
+/datum/surgery_step/mechanic_open
+	name = "unscrew shell"
+	implements = list(
+		/obj/item/screwdriver	= 100,
+		/obj/item/scalpel 		= 75, // med borgs could try to unskrew shell with scalpel
+		/obj/item/kitchen/knife	= 50,
+		/obj/item				= 10) // 10% success with any sharp item.
+	time = 24
+
+/datum/surgery_step/mechanic_open/preop(mob/user, mob/living/carbon/target, target_zone, obj/item/tool, datum/surgery/surgery)
+	user.visible_message("[user] begins to unscrew the shell of [target]'s [parse_zone(target_zone)].",
+		"<span class='notice'>You begin to unscrew the shell of [target]'s [parse_zone(target_zone)]...</span>")
+
+/datum/surgery_step/mechanic_incise/tool_check(mob/user, obj/item/tool)
+	if(implement_type == /obj/item && !tool.is_sharp())
+		return FALSE
+
+	return TRUE
+
+//close shell
+/datum/surgery_step/mechanic_close
+	name = "screw shell"
+	implements = list(
+		/obj/item/screwdriver	= 100,
+		/obj/item/scalpel 		= 75,
+		/obj/item/kitchen/knife	= 50,
+		/obj/item				= 10) // 10% success with any sharp item.
+	time = 24
+
+/datum/surgery_step/mechanic_close/preop(mob/user, mob/living/carbon/target, target_zone, obj/item/tool, datum/surgery/surgery)
+	user.visible_message("[user] begins to screw the shell of [target]'s [parse_zone(target_zone)].",
+		"<span class='notice'>You begin to screw the shell of [target]'s [parse_zone(target_zone)]...</span>")
+
+/datum/surgery_step/mechanic_close/tool_check(mob/user, obj/item/tool)
+	if(implement_type == /obj/item && !tool.is_sharp())
+		return FALSE
+
+	return TRUE
+
+//prepare electronics
+/datum/surgery_step/prepare_electronics
+	name = "prepare electronics"
+	implements = list(
+		/obj/item/device/multitool = 100,
+		/obj/item/hemostat = 10) // try to reboot internal controllers via short circuit with some conductor
+	time = 24
```

Comment:
Why are they making a separate mechanic surgery .dm when they can just put the generic steps into generic_steps.dm? It would be more efficient to do the latter. Mechanic surgery .dm would only be useful for specific surgeries that aren't supposed to be usable on organics.

Reply:
But you can already use cyborg limbs as prosthesis.  Why not just fold the surgery into the same damn file?